otter/integration/tests/test_scaling.py

```diff
@@ -123,3 +119,63 @@ def test_scaling_up(self):
         )
         return d
     test_scaling_up.timeout = 1800
+
+    def test_policy_execution_after_adding_clb(self):
```

Comment:
Can I haz docstring?  :D  This doesn't actually seem to execute the policies yet, just create them?